### PR TITLE
Added mobile responsiveness and some styling. LKI

### PR DESCRIPTION
**Pull request by Leeiaisabelle**
This is for the landing page only.

### CODE REVIEW

**Review by LindaNayeli104 — APPROVED** [review]

**Review by SirEctor — APPROVED** [review]
Should be good, as long as buttons can link to /register and /login.

File: app/templates/index.html [hunk 1]
```diff
@@ -1,34 +1,46 @@
...
+                                    <h1>Feel the Beat!</h1>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col" style="text-align: justify;margin-right: 75px;margin-left: 75px;">
+                                    <p class="text-center" style="padding-right: 50px;padding-left: 50px;text-align: justify;">Track your music according to your mood and open up a world of statistics about your listening habits.<br></p>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-lg-6 text-end mb-4"><button class="btn btn-primary" type="button" style="font-size: 25px;">Register</button></div>
```
Comment (nicolesouydalay) [inline app/templates/index.html]: Link to register.

File: app/templates/index.html [hunk 2]
```diff
@@ -1,34 +1,46 @@
...
+                                <div class="col-lg-6 text-start mb-4"><button class="btn btn-primary" type="button" style="font-size: 25px;">Login</button></div>
```
Comment (nicolesouydalay) [inline app/templates/index.html]: Link to login.

**Review by nicolesouydalay — CHANGES_REQUESTED** [review]
Looks really nice! Just need to add in the links for the buttons.

**Review by nicolesouydalay — APPROVED** [review]

**Pull request merged**